docs/readthedocs/source/doc/PythonAPI/LLM/transformers.rst

```diff
@@ -1,8 +1,14 @@
-BigDL-LLM `transformers`-style API
-=====================
+BigDL-LLM ``transformers``-style API
+====================================
 
-llm.transformers.model
----------------------------
+Hugging Face ``transformers`` Format
+------------------------------------
+
+You can apply BigDL-LLM optimizations on any Hugging Face Transformers models by using the standard AutoModel APIs.
+
+
+AutoModelForCausalLM
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. autoclass:: bigdl.llm.transformers.model.AutoModelForCausalLM
```

Comment:
We need to show the import path as we suggested for users:

```rst
.. autoclass:: bigdl.llm.transformers.AutoModelForCausalLM
```

same for the other auto models.